### DISCUSSION TITLE
How to fully migrate from motioneyeOS to frigate

### DISCUSSION THREAD

**Discussion by Bluscream** [blakeblackshear/frigate#13821] [Ask A Question]
I recently saw the object detection and got impressed, so i snatched a used google coral. Currently im running all my cams (mostly webcams and cheap chinese proprietary PPPP/V380 cams via proxies into a RPi4 4gb running the latest mEOS (6 cams total) and i would like to migrate over. However i couldnt find a single migration tutorial on youtube or google that covers everything. Is frigate just too small a project so youtubers think there is no demand? Or is frigate just too limited to cover all mEOS functionality?

**Comment by NickM-27** [comment]
I'm not familiar with motion eye other than hearing of it, users move from many different projects. The getting started guide should cover how to use frigate including setting up the os and environment

**Comment by arhills** [comment]
I don't think there is going to be a way to migrate i.e. copy over all your config and video files. Frigate is very different in its setup and use. I guess its just a case of starting from scratch with Frigate.
